app/views/users/activate_account.html.erb

```diff
@@ -10,14 +10,25 @@
 </div>
 
 <%- content_for(:no_ember_head) do %>
+  <%= script "preload_store" %>
```

Comment:
This doesn't make sense, just switch it over to the `application` layout, don't recreate the includes in another layout.